config/samples/ovs_v1beta1_ovs.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: ovs.openstack.org/v1beta1
+kind: OVS
+metadata:
+  name: openvswitch
+spec:
+  replicas: 1
+  ovsContainerImage: "quay.io/skaplons/ovs:latest"
+  ovnContainerImage: "quay.io/tripleovictoria/openstack-ovn-controller:current-tripleo"
+  external-ids:
+    system-id: "random"
+    hostname: "devcontainer"
```

Comment:
hostname of pod or may be not required if it's auto picked?

Reply:
Actually I don't think it's needed so I removed it :)